nvm/README.md

```diff
@@ -0,0 +1,34 @@
+Node Version Manager Role
+=========================
+
+Installs NVM on a user by user basis. Can co-exist with a system Node version. 
+
+Config
+------
+
+```yaml
+nvm_config:
+- user: vagrant # username to install NVM for
+  default_version: lts # any version NVM understands to set as default
+  versions: # array of versions to install for user
```

Comment:
Is this not over-complicating things? Do you think a single user will need multiple versions on a provisioned vm?

Reply:
We have that case over at TT- the inspiration for this role over using a system node version.